lib/pure/collections/sequtils.nim

```diff
@@ -1070,18 +1067,18 @@ when isMainModule:
       zip1 = zip(short, long)
       zip2 = zip(short, words)
       zip3 = zip(ashort, along)
-      zip4 = zip(ashort, awords)
-      zip5 = zip(ashort, words)
+      zipXy: seq[tuple[x: int, y: string]] = zip(ashort, awords)
+      zipMn: seq[tuple[m: int, n: string]] = zip(ashort, words)
```

Comment:
I see that you introduce the test here. A comment of why you have this specific assignment is important. I often encounter tests that do things and I never know if something is intentional or not. I often wonder the intention of a test when I read it. Unlike other code, the context of a test does not provide the intention.

Apart from that I think this PR is fine and can be merged.

Reply:
Done!